app/views/layouts/_navigation_links.html.erb

```diff
@@ -1,23 +1,39 @@
-<% if session[:line] %>
-  <ul class="nav navbar-nav navbar-left">
-    <li class="navbar-text"><%= current_line_display %></li>
-    <% if ENV['CM_RESOURCES_URL'] %>
```

Comment:
Yeah, top level, left side, no dropdown. I'd like it to be in the same spot over on the left:

<img width="1207" alt="screen shot 2017-05-06 at 3 29 12 pm" src="https://cloud.githubusercontent.com/assets/3866868/25775338/f28ed1b2-3270-11e7-8614-b1a3017ff3e1.png">